src/routes/users.route.ts

```diff
@@ -0,0 +1,11 @@
+import {createUser, getUsers, getUser, deleteUser, updateUser} from "../controllers/users.controller";

```

Comment:
rename controllers to create, findAll, findOne, deleteOne, updateOne

and import it something like this

import * as UserControllers from '...';

And use it in router like here fastify.post('/', UserControllers.create)